rsts/community/index.rst

```diff
@@ -118,3 +118,332 @@ Flyte maintainers are available on Zoom for you to ask anything, weekly on Wedne
     roadmap
     Frequently Asked Questions <https://github.com/flyteorg/flyte/discussions/categories/q-a>
     troubleshoot
+    
+Flyte Documentation Style Guide
+-------------------------------
+
+Overview
+^^^^^^^^
```

Comment:
```suggestion
*********
Overview
*********
```

Reply:
@trishitapingolia, can you follow 

```
‘#’ for heading 1
‘*’ for heading 2
‘=’ for heading 3
‘-’ for heading 4
‘^’ for heading 5
```

for the headings? Example: https://github.com/benoitbryon/documentation-style-guide-sphinx/blob/master/docs/style-guide.txt